README.md

```diff
@@ -1 +1,38 @@
-# buoy-viewer
+# [Narragansett Bay Data Explorer](https://data-explorer.riddc.brown.edu)
```

Comment:
Other things I'd like to see in the read me is would be link to the live website to the bay explorer.

Reply:
the header is a link to the live website